src/mumble/OverlayConfig.cpp

```diff
@@ -570,5 +586,37 @@ void OverlayConfig::on_qpbSavePreset_clicked() {
 	qs.remove(QLatin1String("usewhitelist"));
 	qs.remove(QLatin1String("blacklist"));
 	qs.remove(QLatin1String("whitelist"));
+	qs.remove(QLatin1String("enablelauncherfilter"));
+	qs.remove(QLatin1String("launchers"));
 	qs.endGroup();
 }
+
+void OverlayConfig::on_qcbEnableLauncherFilter_stateChanged(int state) {
+	qlLauncherFilterDescription->setEnabled(state == Qt::Checked);
+	qlwLaunchers->setEnabled(state == Qt::Checked);
+	qpbAddLauncher->setEnabled(state == Qt::Checked);
+	qpbRemoveLauncher->setEnabled(state == Qt::Checked);
+}
+
+void OverlayConfig::on_qpbAddLauncher_clicked() {
+	QString file = QInputDialog::getText(this, tr("Add launcher"), tr("Add a launcher executable to the Overlay Launcher Filter..."));
```

Comment:
If we manage to place a line edit somewhere we can have the best of both worlds. If not free-text is fine with me. It would be easier to explain. I just have a natural dislike for non-standard file selection ;)